sources/packages/web/src/types/contracts/InstitutionRouteMeta.ts

```diff
@@ -0,0 +1,12 @@
+export enum InstitutionRouteUserType {
```

Comment:
Can you please elaborate on why do we need 2 enums here?
Can they evolve differently in the future?

Reply:
`InstitutionRouteUserType` for router meta, just difference in values with `Institution_` prefix, ya, we can use the same enum